### PR TITLE
Add Chocolatey Installation instructions

### PR DESCRIPTION
**Pull request by gep13**
Based on what is available for Mac, this seems like a good fit.

### CODE REVIEW

**Comment by gep13** [comment]
@FeodorFitsner @IlyaFinkelshteyn what are your thoughts?

@mkevenaar anything else to add here?

File: src/docs/server/index.md [hunk 1]
```diff
@@ -51,6 +51,18 @@ Docker is not required, but recommended! AppVeyor has 1st-class integration with
...
 
+#### Chocolatey Installation
+
+[Install Chocolatey package manager](https://chocolatey.org) if not already installed.
+
+Run the following command:
+
+    choco install appveyor-server
+
+Once AppVeyor service is started, open `http://localhost:8050` in your browser to continue AppVeyor setup.
```
Comment (FeodorFitsner) [inline src/docs/server/index.md]: This line is not needed as browser window will be opened automatically as with manual installation. I've just checked. Maybe we should add the same wording as below.

**Comment by FeodorFitsner** [comment]
Absolutely, choco installation instructions have been missing! :) I left one comment though.

**Comment by gep13** [comment]
@FeodorFitsner that makes sense!  I have updated the PR.

File: src/docs/server/index.md [hunk 2]
```diff
@@ -51,6 +51,18 @@ Docker is not required, but recommended! AppVeyor has 1st-class integration with
...
 AppVeyor Server can be installed on **Windows 7 SP1 or later** and **Windows Server 2008 R2 SP1 or later**.
 
+#### Chocolatey Installation
+
+[Install Chocolatey package manager](https://chocolatey.org) if not already installed.
+
+Run the following command:
+
+    choco install appveyor-server
+
```
Comment (mkevenaar) [inline src/docs/server/index.md]: I would go with something like this:
```powershell

They do the same in the rest of the document for e.g. yaml

**Comment by mkevenaar** [comment]
I have made one comment. But looks good to me!

**Comment by FeodorFitsner** [comment]
Hi guys,

a new 7.0.2353 version has been just released.

Some notable changes:

- It's now called "Team Edition" and includes **unlimited** (yes!) user accounts for free
- Copy on https://www.appveyor.com/on-premise/ has been updated too - you may want to reflect that in package description.

Could you please update Chocolatey package, so I can merge this PR?

Thank you!

**Comment by gep13** [comment]
@mkevenaar are you able to make the changes to the Package, or would you like some help?

**Comment by mkevenaar** [comment]
@gep13 help is always welcome. I will not be able to do that just now. Do you want to submit a PR?

**Comment by mkevenaar** [comment]
I have created an issue for this change and will submit a new package out soon. https://github.com/mkevenaar/chocolatey-packages/issues/27

**Comment by mkevenaar** [comment]
@FeodorFitsner @gep13 The documentation change is committed, an appveyor build is running to submit the latest package and will be available soon on chocolatey.org.

**Comment by gep13** [comment]
@FeodorFitsner is this ready to go?

**Comment by FeodorFitsner** [comment]
I'm thinking maybe it makes more sense to move Chocolatey instructions under "Manual installation" as there will be moments when a newer MSI released but Chocolatey package is not updated yet. Right?

**Comment by mkevenaar** [comment]
> there will be moments when a newer MSI released but Chocolatey package is not updated yet. Right?

My build runs every 6 hours, and usually it takes less than 1h to process the verification on chocolatey.org. If you want, a badge can be added with the current version: 

[AppVeyor Server (Team Edition))](https://chocolatey.org/packages/appveyor-server)

`[AppVeyor Server (Team Edition))](https://chocolatey.org/packages/appveyor-server)`

**Comment by FeodorFitsner** [comment]
Sounds great, thank you!

**Pull request merged**